l10n_it_fatturapa_out_triple_discount/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Italian Localization - Fattura elettronica - '
```

Comment:
Il nome non dovrebbe iniziare con "ITA - Fattura elettronica ....."?

Reply:
Fixed in https://github.com/OCA/l10n-italy/compare/ba555e87e3cb3289babe79c1c656be0c6c51d4e1..f94bff2fabf3d89fb65a4cbb48f5f39b28e42829 Thanks.